app/assets/javascripts/components/InvitationForm.coffee

```diff
@@ -11,12 +11,17 @@ invitationForm = React.createClass
 
   submit: (enteredValues) ->
     { dispatch } = Screensmart.store
-
-    # Place domainId value in an array. To be removed in the future when multiple domains
-    # can be chosen.
-    enteredValues.domainIds = [enteredValues.domainIds]
     dispatch Screensmart.Actions.sendInvitation(enteredValues) if @props.valid
 
+  toggleDomain: (domainId, checked) ->
+    { fields: { domainIds } } = @props
+    if checked
+      domainIds.addField(domainId)
```

Comment:
Bedankt voor de uitleg. Zo ver was ik nog niet in de code gedoken. Ik zal mij eens gaan verdiepen in redux en redux-form.